### DISCUSSION TITLE
Lost page

### DISCUSSION THREAD

**Discussion by codejoy** [requarks/wiki#7477] [Help / Questions]
I manage a wikijs rollout (to replace our old one) and today I woke up to an email that the users using it, lost a page, or its no longer there....  I have the URL for it, and sure enough going to it a big "Page not found... + create page". web page comes up.  I am not sure what or how this got lost, though if I understand correctly wikijs is not a flat file system.   IS there anyway I can find out what might of happened to it, did a user delete it?  Is it gone for ever? This runs as a VM and I do back it up via Veeam and can use the sledge hammer and recover to the last full backup, but if I can just find this one page, that would be ideal...  not sure what to try besides the sledgehammer approach.

**Comment by codejoy** [comment]
So one of the users showed me they edited a page and the missing page is all there now under a new page. I am not sure what they did or how this happened, so I guess my real question is there anywhere that the steps users are taking is logged?   I want to recreate this issue so I know how to recover from it.   I am at a lost (admiringly, I deployed this app for them and don't use it much).

**Comment by NGPixel** [comment]
The page was likely renamed / moved?
